### ISSUE TITLE
Access User Location by user-id, app-id and app-secret

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kalyanhadooptraining/facebook-java-api#350]
```
If  my username is 'pavan' and if someone elses username who is not my friend 
is 'jason'  . If I login from my account and type facebook.com/jason, I can see 
some of his details like his current place, his birthday if he has made them 
public. Likewise in my application I have some bunch of userids and usernames, 
I have my appid and appsecret.. So using my appID , AppSecret and userids, can 
I get users location or the data about that username (which I can see when I 
login from my account(pavan)  and and type facebook.com/{username}).

   In other words, I cant go to them and ask them all to grant permissions for my application for location. I want to use just my appID and appSecret and their userID to get their location info if he has made that public.

Is there a way to acheive this?

If it is not possible in JAVA, Is it possible in PHP? Any help is appreciated.

```

Original issue reported on code.google.com by `neehar87...@gmail.com` on 20 Aug 2011 at 5:52
